spartan/aztec-network/values/exp-2.yaml

```diff
@@ -1,34 +1,338 @@
+    - 0xE77478D9E136D3643cFc6fef578Abf63F9Ab91B1
+    - 0x6C8EA11559DFE79Ae3dBDD6A67b47F61b929398f
+    - 0x48fA7b63049A6F4E7316EB2D9c5BDdA8933BCA2f
+    - 0x16aDfbeFdEfD488C992086D472A4CA577a0e5e54
+    - 0x225356FF5d64889D7364Be2c990f93a66298Ee8D
+    - 0xcBDc0F9a4C38f1e010bD3B6e43598A55D1868c23
+    - 0xBc5BdceE96b1BC47822C74e6f64186fbA7d686be
+    - 0x0536896a5e38BbD59F3F369FF3682677965aBD19
+    - 0xFE0f143FcAD5B561b1eD2AC960278A2F23559Ef9
+    - 0x98D08079928FcCB30598c6C6382ABfd7dbFaA1cD
+    - 0x8C3229EC621644789d7F61FAa82c6d0E5F97d43D
+    - 0x9586A4833970847aef259aD5BFB7aa8901DDf746
+    - 0x0e9971c0005D91336c1441b8F03c1C4fe5FB4584
+    - 0xC4c81D5C1851702d27d602aA8ff830A7689F17cc
+    - 0x9c79357189d6aF261691ECF48de9A6bbF30438Fc
+    - 0xd96Eb0f2e106eA7c0a939e9C460a17ace65FeCFF
+    - 0x4548774216F19914493d051481FEb56246bC13f0
+    - 0xFdAA62Ea18331AFA45CC78b44DBa58d809EaB80e
+    - 0x7D19cEa5598AccbBF0005A8EB8ed6A02C6F8aB84
+    - 0xEaBd5094570298Ffd24e93E7af378162884611cb
+    - 0x51953940F874eFa94F92EB2d6Aed023617A07222
+    - 0x6813ae1FC15E995230C05D4480d50219Bb635F15
+    - 0x11c9CFEc77102a7C903a2D2319c79E7b0BBC9235
+    - 0xBE9086f1A38740F297F6347B531732541289b220
+    - 0xD4Db664B707353422b1Ffc94038cDD0a7D074D51
+    - 0x11Ba29fE987adDFa480FfEcf3d98B26630917a78
+    - 0xFFd57510605b4F47A58576ccC059aB8882C7eA00
+    - 0x83781Cf2371117aAc856621805FB83c9ca439BAd
+    - 0x2Bac2e5a4f39c32eD16205591Ba26e307414cA9E
+    - 0x8D86EF40df93B1b3822bf996B972bA53E79C07c9
+    - 0xc9DB8bec097C2CbdcE109e03A36f98a87E04ffEf
+    - 0x925EB78bEb9a84cf88dc92DF5f2FE7bF33b40104
+    - 0x5061a633977476e18B99423Ac51C6Df50621a597
+    - 0x46655f18cC741152515f0d843AD4355b4AD23377
+    - 0x012af9b68f94FE212705B708Cb69b53508D522a2
+    - 0x87194239f32f33d74C99B2545032623B32e3F795
+    - 0x0149aF4613AA457765d23a38D62685a9623AEf35
+    - 0xBCcA9EcB933Db2481111102E73c61C7c7C4e2366
+  resources:
+    requests:
+      memory: "5Gi"
+      cpu: "1.5"
+      ephemeral-storage: "275Gi"
+  maxOldSpaceSize: "4608"
   validator:
     disabled: false
+  sequencer:
+    maxTxsPerBlock: 40
+    enforceTimeTable: true
 
 bootNode:
-  seqPublisherPrivateKey:
+  peerIdPrivateKey: 080212200ba8451c6d62b03c4441f0a466c0bce7a3a595f2cf50a055ded3305c77aa3af0
+  storageSize: "100Gi"
   validator:
     disabled: true
+  resources:
+    requests:
+      memory: "5Gi"
+      cpu: "1.5"
+      ephemeral-storage: "275Gi"
+  maxOldSpaceSize: "4608"
+
+proverAgent:
+  replicas: 8
+
+proverBroker:
+  resources:
+    requests:
+      memory: "12Gi"
+      cpu: "3.5"
+      ephemeral-storage: "275Gi"
+  maxOldSpaceSize: "11776"
 
 proverNode:
-  proverPublisherPrivateKey:
+  resources:
+    requests:
+      memory: "12Gi"
+      cpu: "3.5"
+      ephemeral-storage: "275Gi"
+  maxOldSpaceSize: "11776"
 
 bot:
-  txIntervalSeconds: 20
+  replicas: 10
+  followChain: "NONE"
+  enabled: true
+  txIntervalSeconds: 1
+
+jobs:
+  deployL1Verifier:
+    enable: false
+
+ethereum:
```

Comment:
This will probably work, but the resources tab for ethereum has been divided into the 3 clients 

execution:
    resources:
    
beacon:
    resources:
    
validator:
    resources